.changeset/angry-pandas-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/vertexai': minor
```

Comment:
This is somewhat of an unusual situation, but before I/O, we don't need changesets for PRs against the feature branch. Once the feature branch is merged to master, we will, so you may need this if we merge this after we merge the feature branch for I/O. Depends on the timing.

Reply:
SG, I removed the changeset